Debian/bin/lib/Cyrus/Docker/Command/build.pm

```diff
@@ -17,6 +17,7 @@ sub abstract { 'configure, build, and install cyrus-imapd' }
 sub opt_spec {
   return (
     [ 'recompile|r', 'recompile, make check, and install a previous build' ],
+    [ 'no-cunit|n',  'do not run make check' ],
```

Comment:
`no-X` options should use the `!` modifier, so:

```perl
    [ 'cunit!', "run make check", { default => 1 } ],
```

Then `--no-cunit` works.  That won't get you `-n`, though.  So you want:

```perl
    [ 'cunit!', "run make check", { default => 1 } ],
    [ 'n', "hidden", { implies => { cunit => 0 } } ],
```

Yes, I really think it's worth it to keep toggles using `!` for consistency. :)

Reply:
I've made this change. I don't quite see how it's valuable beyond "Well you can be explicit if you want and say `--cunit` even if that's the default...`. I've modified your recommendation slightly so the `-n` is visible in the help

Reply:
Just consistency.  Matter of taste, at some level, but:  on/off flags should be on/off flags, and anything no- should have a matching yes- (well, no-no (well, "")).